app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ActionMailer::Base
+  default from: 'info@stavcdflashcard.com'
+
+  def welcome_email(user)
+    @user = user
+    @url =  'http://stavcdflashcards.herokuapp.com/login'
+    mail(to: @user.email, subject: 'Welcome to Stavcd Flashcards')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.